worker/deploy/session-memory/entrypoint.sh

```
@@ -1,34 +1,59 @@
+positive_int_or_default() {
+  local value="${1:-}"
+  local default_value="$2"
+  if [[ "$value" =~ ^[0-9]+$ ]] && [ "$((10#$value))" -gt 0 ]; then
+    printf '%s\n' "$((10#$value))"
+  else
+    printf '%s\n' "$default_value"
+  fi
+}
+
+main() {
+  cd /app
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)